### ISSUE TITLE
API GM_download can't use with Firefox browser

### ISSUE THREAD

**Issue by tick97115115** [Tampermonkey/tampermonkey#949]
(Please fill out the issue template with your details)
I was trying to download some blob object, but GM_donwload doesn't work.

- I has been adjust the download option to "browser API"
- saveAs setting :true

## Expected Behavior
- download window popup

## Actual Behavior
- Not any download window popup or download mission start.

## Specifications

- Firefox: (76.0.1)
- TM: (4.10.6112)
- OS: (Windows10 18363.836)

## Script
```js
// ==UserScript==
// @name MyName
// @namespace http://mywebsite.com
// @Version 1.0
// @description try to take over the world!
// @author Me
// @match http://mywebsite.com/*
// @require http://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js
// @include https://mywebsite.com/*
// @grant GM_download
// ==/UserScript==

let text = "String";
let data = new Blob([text],{type:'text/plain'});
let arg = {
    url: URL.createObjectURL(data),
    name: "textInfo.txt",
    saveAs: true,
}
GM_download(arg)
```

**Comment by tick97115115** [comment]
An additional info:
The GM_download API was worked fine with Edge browser.

**Comment by mkckr0** [comment]
I also have this problem.

**Comment by derjanb** [comment]
Should be fixed at TM BETA 4.11.6113 (https://www.tampermonkey.net/index.php?browser=firefox)
